docs/templating/templates/view_matcher_reference.md

```diff
@@ -217,3 +222,39 @@ match:
 match:
     '@Ibexa\Contracts\ProductCatalog\ViewMatcher\LocationBased\RootLocation': true
 ```
+
+## Taxonomy entry ID
+
+`Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Id` matches based on an ID of the taxonomy entry.
+
+``` yaml
+match:
+    '@Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Id': [1, 2, 3]'
+```
+
+## Taxonomy entry identifier
+
+`Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Identifier` matches based on an identifier of the taxonomy entry.
+
+``` yaml
+match:
+    '@Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Identifier': ['spring', 'events', 'devices']
+```
+
+## Taxonomy entry level
+
+`Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Level` matches based on a level of the taxonomy entry.
```

Comment:
```suggestion
`Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Level` matches based on a level of the taxonomy entry.
It can match on one level, on level in a list, or on level constraint with one of the following operators: `<`, `>`, `<=`, `>=`.
```